lib/cli/utils.js

```diff
@@ -88,7 +88,8 @@ utils.iterativeCompletion = (inputs, depth, completer) => {
   // they'll be not valid (those are completions for previous inputs)
   if (newDepth <= inputs.length - 1) completions = [];
   if (!completions[0] ||
-      completions.length === 1 && completions[0] === inputs[depth]) {
+    completions.length === 1 && completions[0] === inputs[depth]
```

Comment:
I mean if we align condition expression like @belochub posted we need to see below how to allign code inside `if`, so I showed in the first part of comment that it is not readable and proposed different alignment @lundibundi